ruuvigw2mqtt/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "main": "index.js",
+  "dependencies": {
+    "mqtt": "4.2.8",
+    "ruuvitag-parser": "0.2.9"
```

Comment:
Now I'm using the `RuuviTag-Parser` I created with TypeScript and it seems to be working as intended 🙂